docs/notebooks/4.1 Setting up and Using PV Sites.ipynb

```diff
@@ -0,0 +1,521 @@
+      "text/plain": [
+       "<Figure size 1500x400 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "resource_id = metadata[\"resource_id\"]\n",
+    "timeseries = {}\n",
+    "\n",
+    "# NOTE: this will consume an API request!\n",
+    "res = live.advanced_pv_power(resource_id)\n",
+    "timeseries[\"initial\"] = res.to_pandas()\n",
+    "timeseries[\"initial\"].plot(figsize=[15, 4])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "daf70ddc-f878-4df5-a04c-1059cc86a77d",
+   "metadata": {},
+   "source": [
+    "We can adjust metadata to our site using `pv_power_sites.patch_pv_power_site`.  You can see definitions of the various available parameters in the [API documentation](https://docs.solcast.com.au/#ba412164-31c2-47a9-a965-c95bc9b632a6).  Let's say we know our site has bifacial panels and that the tracker axis is oriented 10 degrees from north towards the east.  We can adjust the site's settings accordingly."
```

Comment:
We can adjust metadata to our site using `pv_power_sites.patch_pv_power_site`
This doesn't make sense to me, maybe:
"We can update the metadata of our using...


`[API documentation](https://docs.solcast.com.au/#ba412164-31c2-47a9-a965-c95bc9b632a6)` these links aren't static so you can't use the hash link
might also want to point them at the Advanced PV webpage again as it also includes definitions and explanations https://solcast.com/advanced-pv-power-model

`We can adjust the site's configuration accordingly.` Just to avoid using settings, metadata, and configuration to refer to the same thing

Reply:
I've avoided the use of the word 'update' as there is another endpoint update_pv_power_site which replaces everything in the configuration, rather than patching specific items.   Have changed link and removed 'metadata'.